package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.16.5",
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍